torch/testing/_internal/common_utils.py

```diff
@@ -577,8 +577,18 @@ def run_tests(argv=UNITTEST_ARGS):
         failed_tests = []
         for case in test_cases:
             test_case_full_name = case.id().split('.', 1)[1]
-            exitcode = shell([sys.executable] + argv + [test_case_full_name])
+            cmd = [sys.executable] + argv + [test_case_full_name]
+            string_cmd = " ".join(cmd)
+            exitcode = shell(cmd)
             if exitcode != 0:
+                # This is sort of hacky, but add on relevant env variables for distributed tests.
+                if 'TestDistBackendWithSpawn' in test_case_full_name:
```

Comment:
Should we stop testing spawn altogether? I think in the internal tests we already removed spawn right?